.github/workflows/codeql-push.yml

```diff
@@ -0,0 +1,55 @@
+on:
+  push:
```

Comment:
If you need to filter this further so it only runs when there are changes on the components you're building on push, you could do so with this syntax: https://help.github.com/en/actions/reference/workflow-syntax-for-github-actions#onpushpull_requestpaths. Otherwise, LGTM!

Reply:
Thanks for the link! That's great. In a follow-up I'll start making push workflows for different sub-components when there are changes there.